providers/tests/amazon/aws/log/test_s3_task_handler.py

```diff
@@ -128,7 +128,10 @@ def test_read(self):
         ti.state = TaskInstanceState.SUCCESS
         log, metadata = self.s3_task_handler.read(ti)
         actual = log[0][0][-1]
-        expected = "*** Found logs in s3:\n***   * s3://bucket/remote/log/location/1.log\nLog line"
+        expected = " INFO - ::group::Log message source details\n"
+        expected += "*** Found logs in s3:\n***   * s3://bucket/remote/log/location/1.log\n"
+        expected += " INFO - ::endgroup::\n"
+        expected += "Log line"
```

Comment:
...so for example like:
```suggestion
        assert "*** Found logs in s3:\n***   * s3://bucket/remote/log/location/1.log" in actual
        assert "Log line" in actual
```